manifests/plugin/sensors.pp

```diff
@@ -1,21 +1,27 @@
 # https://collectd.org/wiki/index.php/Plugin:Sensors
 class collectd::plugin::sensors (
-  $ensure           = 'present',
-  $manage_package   = undef,
-  $sensorconfigfile = undef,
-  $sensor           = undef,
-  $ignoreselected   = undef,
-  $interval         = undef,
+  $ensure                  = 'present',
+  $manage_package          = undef,
+  $sensorconfigfile        = undef,
+  $sensor                  = undef,
+  $ignoreselected          = undef,
+  $interval                = undef,
+  $package_install_options = $collectd::package_install_options,
 ) {
 
   include ::collectd
 
   $_manage_package = pick($manage_package, $::collectd::manage_package)
 
+  if $package_install_options != undef {
+    validate_array($package_install_options)
```

Comment:
It is best practice to define the array with all arguments. You don't know the amount of options used however.
Optional[Array[String[]]] $package_install_options   = undef
will allow only any number of elements in the array.
Can you please advise if this is allowed ?It does not seem to be the way to go according to several sources.